addons/l10n_in/models/res_partner.py

```diff
@@ -2,16 +2,55 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import api, fields, models, _
-from odoo.exceptions import ValidationError
 
 
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
-    # Use in view attrs. Need to required state_id if Country is India.
-    l10n_in_country_code = fields.Char(related="country_id.code", string="Country code")
+    l10n_in_gst_treatment = fields.Selection([
+        ('regular', 'Registered Business - Regular'),
+        ('composition', 'Registered Business - Composition'),
+        ('unregistered', 'Unregistered Business'),
+        ('consumer', 'Consumer'),
+        ('overseas', 'Overseas'),
+        ('special_economic_zone', 'Special Economic Zone'),
+        ('deemed_export', 'Deemed Export'),
+    ],string="GST Treatment")
+    l10n_in_pan_number = fields.Char(string="PAN Number")
```

Comment:
PAN number is for unregistered persons who don't have GSTIN.
We can't put PAN number in vat field because when Invoice or any document is print(PDF) then it's print with label GSTIN